### PR TITLE
Add skeleton of the project

### PR DESCRIPTION
**Pull request by cmolinan**
### Added Changes
- Add the toolbar section 
- Add the header section 
- Add a .gitignore file
- Set up the linter for HTML and CSS
- Add descriptive README

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <title>Portfolio: setup and mobile version skeleton</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <main class="container">        
+        <header>
+            <div class="toolbar">
+                <h2>My Logo</h2>            
+                <img src="images/union.svg" alt="menu" />
```
Comment (aboongm) [inline index.html:15]: - The logo should have a tag link. This is going to become your logo, so plan it in such a way.
- The hamburger menu should be inside a nav tag

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  padding-right: 17px;
+  height: fit-content;
+}
+
+.toolbar {
+  height: auto;
+  width: auto;
+  display: flex;
+  align-items: center;
+}
```
Comment (aboongm) [inline styles.css:83]: - toolbar class for the header section should use flexbox. Using the flex will help you in controlling the items viz the logo and the navigation items which you are yet to put.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+.container {
+  display: flex;
+  border: 1px solid blue;
+  width: 375px;
+  height: 100vh;
+  flex-wrap: wrap;
+  flex-direction: column;
+  background-image: url("images/header-shapes mobile@2x.svg");
+  background-repeat: no-repeat;
```
Comment (aboongm) [inline styles.css:17]: - You have applied the background-image in the container class on the main section. This should be preferably applied to the headline class. It depends on how you want to structure your future code viz portfolio section, about-me section, and contact me section. If you intend to put those sections inside the main tag with the container class, you will have a problem.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        <header>
+            <div class="toolbar">
+                <h2>My Logo</h2>            
+                <img src="images/union.svg" alt="menu" />                 
+            </div>
+        </header>
+
+        <section class="headline">
+            <h2><br>I'm Carlos<br>Glad to see<br>you!</h2>            
+            <p>I’m a software developer! I can help you<br>build a product , feature or website Look<br>through some of my work and<br>experience! If you like what you see and<br>have a project you need coded, don’t<br>hestiate to contact me.</p>
```
Comment (aboongm) [inline index.html:21]: - Your h2 tag with your main title should be an h1 tag

File: index.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+            <h3>LET'S CONNECT</h3>
+              <div class="icons-social">
+                <img src="images/twitter.png" class="social" alt="twitter" />            
+                <img src="images/Linkedin.png" class="social" alt="Linkedin" />
+                <img src="images/medium.png" class="social" alt="Medium" />
+                <img src="images/github.png" class="social" alt="Github" />            
+                <img src="images/angellist.png" class="social" alt="angellist" />
+              </div>
```
Comment (aboongm) [inline index.html:29]: - Your list of icons should use a ul tag to group them, and li tag per icon. Each icon should have a link (a tag)

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+  background-size: auto;
+  background-position: center;
+  background-color: white;
+}
+
+.headline {
+  height: auto;
+  width: auto;
+  margin-top: 120px;
+}
```
Comment (aboongm) [inline styles.css:27]: - Your headline class inside headline section should use flexbox to be able to control the contents inside.
- Follow the figma design for fonts, font-size, and color. The margin/padding should feel similar to the figma design.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: setup and mobile version skeleton</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <main class="container">        
+        <header>
```
Comment (aboongm) [inline index.html:12]: port-1

Please compare your design and the figma and see the difference.
1. There is a margin around your content. Try if applying margin of 0 to body remove the space
2. You have set the width to 375px in container class and this is preventing your website from being responsive. You have to set the max-width to 100% or vw
3. The background image is not the same as in figma design
4. The font of your main title is not the same as in the figma design.  Everywhere its imperative that you follow the figma design for fonts, font-size, and color. The margin/padding should feel similar to the figma design.

**Review by aboongm — CHANGES_REQUESTED** [review]
Hi @cmolinan , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

request_changes_1

## Highlights
- All linters are passing ✔️
- Good commit history ✔️
- Good PR description ✔️

## Required Changes ♻️

_Please check the comments below for the required changes._

## Optional suggestions

[OPTIONAL] Please update the README. Check comments below

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 8]
```diff
@@ -1,73 +1,21 @@
...
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
```
[the remark this answers is not in the document]
Comment (aboongm) [reply README.md:36]: @cmolinan The README update is optional. You can do it at the last. The other required changes should be implemented as they are part of the project requirements. Please let me know where you are having problems.

**Review by Ismailco — APPROVED** [review]
### Hi @cmolinan,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**